src/domains/issue/controller/issue.controller.ts

```diff
@@ -34,6 +36,14 @@ issueRouter.get('/:issueId/pause', validateRequest(IssuePauseParams, 'params'),
   return res.status(HttpStatus.OK).json(event);
 });
 
```

Comment:
Just a comment. I was thinking that as your are making changes in database, the resume and pause endpoint should have another verb (post, put)